tests/test_derivatives.py

```diff
@@ -165,16 +165,19 @@ def test_fd_space(derivative, space_order):
 
 @skipif_yask
 @pytest.mark.parametrize('space_order', [2, 4, 6, 8, 10, 12, 14, 16, 18, 20])
+@pytest.mark.parametrize('deriv_order', [1, 2])
 @pytest.mark.parametrize('stagger', [centered, right, left])
 # Only test x and t as y and z are the same as x
-def test_fd_space_staggered(space_order, stagger):
+def test_fd_space_staggered(space_order, stagger, deriv_order):
     """
     This test compares the discrete finite-difference scheme against polynomials
     For a given order p, the finite difference scheme should
     be exact for polynomials of order p
     :param derivative: name of the derivative to be tested
     :param space_order: space order of the finite difference stencil
     """
+    if space_order == 2 and deriv_order == 2:
+        return True
```

Comment:
?

Reply:
Stagerred FD only has 2 points for this case wich is not enough for 2nd order FD (needs at least 3)

Reply:
but it does when ``deriv_order == 1``?

Reply:
Yes `deriv_order == 1` is fine with 2 points